e2e/cypress/support/api/user.d.ts

```diff
@@ -340,5 +356,15 @@ declare namespace Cypress {
          *   cy.apiUpdateUserAuth('user-id', 'auth-data', 'password', 'auth-service');
          */
         apiUpdateUserAuth(userId: string, authData: string, password: string, authService: string): Chainable<Response>;
+
+        /**
+         * Update a user's auth data
+         * See https://api.mattermost.com/#tag/users/paths/~1users~1{user_id}~1mfa/put
```

Comment:
Looks like the correct link is `https://api.mattermost.com/#tag/users/paths/~1users~1{user_id}~1auth/put`